know-your-computer/readme.md

```diff
@@ -0,0 +1,26 @@
+# Know Your Computer
+
+"Know-Your-Computer" focuses on how to perform regular tasks on your computer with the use of the terminal using basic linux commands. This will help reduce the complexity and fear of using the terminal. 
+
+## Pre-requisite
+To be able to do this workshop you will require a laptop that has a linux shell running on it. If you already have a Mac or an Ubuntu operating system or any other linux distribution running on your laptop, then you are fine. If your laptop is running on Windows operating system, it is recommended that you download and install git bash for windows using this [link](https://git-scm.com/downloads).
+Once installed, open git bash and you should be able to perform the tasks below. 
+
+## Tasks
+ - Create a folder called my-shopping inside your Documents folder
+ - Then create a file called my-list.txt inside my-shopping folder
+ - Write the following items to my-list.txt
+   1. Tea
+   2. Pasta
+   3. Coffee
+   4. Milk
+- Create a new folder called my-new-shopping inside your Documents folder
+- Now move my-list.txt that is already inside my-shopping folder into my-new-shopping folder so that it does not exist anymore my-shopping folder
+- Make a copy of the my-list.txt that is inside my-new-shopping folder back to my-shopping folder
+- Now add
+   5. Sugar to the my-list.txt that is inside my-new-shopping.
+
+## Resources
+To help with this here are resources you can refer to:
+[freecodecamp.com](https://www.freecodecamp.org/news/the-linux-commands-handbook/)
+[Guru99](https://www.guru99.com/linux-commands-cheat-sheet.html)
```

Comment:
[CYF Treasure Hunt](https://github.com/CodeYourFuture/CLI-Treasure-Hunt)

Reply:
I've added the changes :)